pom.xml

```diff
@@ -8,10 +8,17 @@
     <artifactId>doc-samples</artifactId>
     <packaging>pom</packaging>
 
+    <!-- Parent POM defines common plugins and properties. -->
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>shared-configuration</artifactId>
+        <version>1.0.0</version>
+        <relativePath>java-repo-tools</relativePath>
+    </parent>
+
```

Comment:
I mailed https://github.com/GoogleCloudPlatform/java-docs-samples/pull/129 to do this in a separate PR.